x-pack/plugins/secops/public/components/page/network/network_top_n_flow_table/select_type.tsx

```diff
@@ -12,40 +12,47 @@ import { NetworkTopNFlowDirection, NetworkTopNFlowType } from '../../../../graph
 
 import * as i18n from './translations';
 
-const toggleTypeOptions = [
+const toggleTypeOptions = (id: string) => [
```

Comment:
Is it possible to type this array and add it here as in:

```ts
const toggleTypeOptions = (id) : MyBrandSpankingNewType => [
```

Is there an existing type we can use to help us out for options?

Reply:
I really do think that EUI should have it as a type and not us creating type for EUI